v2/tools/asoctl/internal/importing/importable_arm_resource.go

```diff
@@ -60,37 +71,161 @@ func (i *importableARMResource) Resource() genruntime.MetaObject {
 }
 
 func (i *importableARMResource) Import(ctx context.Context) ([]ImportableResource, error) {
```

Comment:
I _just_ realized that the resources returned here aren't imported yet, they're just child resources to be imported. I do still think my "do we want a mode where we only import the one resource ID" applies, but probably my confusion about thinking this could be called `ImportAll` would be resolved by putting the `Import` interface comment here too.

Reply:
Added a comment.